### DISCUSSION TITLE
Error al intentar instalar ipywidgets delMódulo 0

### DISCUSSION THREAD

**Discussion by GibsDevs** [LaunchX-InnovaccionVirtual/CursoIntroPython#308] [Q&A]
Revisando todas las discusiones previas, no encontré un error similar al mío, y no puedo avanzar :(
Vean los "pantallazos" y lo que se menciona al final:
![image](https://user-images.githubusercontent.com/44006640/154637846-78346fd8-f34b-4511-b264-17031f1a33d7.png)
![image](https://user-images.githubusercontent.com/44006640/154637988-fd526ebf-5c28-4608-ad35-cea240b52ff6.png)

Ya con eso no puedo seguir avanzando. ¿Alguien que pueda orientar?
Gracias

**Comment by JavierMendez-Coder** [comment]
Buenas noches.
Parece ser que tu versión de `pip` no se encuentra actualizada a la versión actual. Puedes revisar la versión de `pip` instalada en tu computadora desde la terminal con el comando:
```
pip --version
```
En caso de contar con una versión desactualizada, corre este otro commando para actualizarlo:
```
pip install --upgrade pip
```
Como referencia la versión actual de `pip` es la: `v22.0.3`.
<br>
Espero haberte ayudado. 👋

**Reply by GibsDevs** [reply]
La versión de pip es la más reciente: 
![image](https://user-images.githubusercontent.com/44006640/154716656-315480c2-28e4-4b1f-b3a7-5bc4dc5f92fa.png)

**Reply by JavierMendez-Coder** [reply]
La advertencia que te aparece en la consola, te dirige hacia un `issue` de GitHub donde comentan que, puede aparecer por un intento de actualizar `pip`, que ha resultado típicamente en (partes de) múltiples versiones de `pip` siendo instaladas en una misma instalación de Python.
Puede ser que esto sea la fuente de tu problema con `pip` a la hora de instalar el paquete. No obstante, no estoy 100% seguro.
Puedes tratar de forzar la reinstalación de `pip` con el siguiente comando:
```
python -m pip install --upgrade --force-reinstall pip
```
De igual forma te recomiendo pasarte por el issue y revisar las respuestas de otros usuarios, a lo mejor puedes encontrar algo que te pueda servir.

**Comment by Ingrzs** [comment]
Fallo al generar el metadato intenta instalar nuevamente, si sucede lo mismo inténtalo con %pip

**Reply by GibsDevs** [reply]
Ya lo había intentado varias veces, y seguía el mismo error.
Ahora, al intentar con %pip, sigue igual:
![image](https://user-images.githubusercontent.com/44006640/154746676-f16fee37-1809-4dc5-b60c-ccd6f16710bb.png)
![image](https://user-images.githubusercontent.com/44006640/154746737-a6c6cbcf-6fc3-4ebb-a3b5-6a63ec707034.png)
